app/controllers/houses_controller.rb

```diff
@@ -61,7 +61,7 @@ def current_parties
 
     @house, @parties = data.filter('http://id.ukpds.org/schema/House', 'http://id.ukpds.org/schema/Party')
     @house = @house.first
-    @parties = @parties.sort_by(:name)
+    @parties = @parties.sort_by(:member_count).reverse
```

Comment:
I've implemented this in parliament ruby and I am now using it in the parliament.uk prototype.